main.go

```diff
@@ -3,69 +3,53 @@ package main
 import (
 	"flag"
 	"fmt"
-	"os"
-	"sync"
-	"time"
-
-	"github.com/golang/glog"
 	"github.com/mesosphere/mesos-dns/logging"
 	"github.com/mesosphere/mesos-dns/records"
 	"github.com/mesosphere/mesos-dns/resolver"
-
-	"github.com/miekg/dns"
+	"os"
+	"sync"
+	"time"
 )
 
 func main() {
-	var wg sync.WaitGroup
 	var resolver resolver.Resolver
+	var versionFlag bool
+	var wg sync.WaitGroup
 
-	versionFlag := false
-
-	cjson := flag.String("config", "config.json", "location of configuration file (json)")
+	// parse flags
+	cjson := flag.String("config", "config.json", "path to config file (json)")
```

Comment:
might be nice to implement something like this:
- /path/to/file
- file:///path/to/file
- http://host:port/path/to/file/on/web/server
... etc